test/Fail/Issue1944-instance.err

```diff
@@ -1,4 +1,8 @@
 Issue1944-instance.agda:23,14-20
-Cannot resolve overloaded projection test because principal
-argument is not of record type
+Cannot resolve overloaded projection 'test' because
+principal argument 'x' has type 'A'
+ while it should be of record type
+Signatures in the scope: 
+- test : ({A = A₁ : Set} ⦃ r : Testable A₁ ⦄ → A₁ → Bool)
+- test : ({@0 A = A₁ : Set} → Testable A₁ → A₁ → Bool)
```

Comment:
Could the new "Signatures in scope" part maybe be indented (`nest 2`)?  It breaks a bit the flow of its surrounding sentence, so maybe indentation would mark it as "bracketed phrase".

Reply:
Sure.